tests/platform_tests/conftest.py

```diff
@@ -149,20 +166,22 @@ def service_time_check(message, status):
             datetime.strptime(timestamps["Starting"], FMT)).total_seconds() \
                 if "Started" in timestamps and "Starting" in timestamps else None
 
-        timings["reboot_time"] = (datetime.strptime(timestamps["Started"], FMT) -\
-            datetime.strptime(timestamps["Stopped"], FMT)).total_seconds() \
-                if "Started" in timestamps and "Stopped" in timestamps else None
-
-    files = glob.glob('/tmp/*-report.json')
-    if files:
-        filepath = files[0]
-        with open(filepath) as json_file:
-            report = json.load(json_file)
+        if "Started" in timestamps and "Stopped" in timestamps:
+            timings["reboot_time"] = (datetime.strptime(timestamps["Started"], FMT) -\
+                datetime.strptime(timestamps["Stopped"], FMT)).total_seconds()
+        elif "Start" in timestamps and "End" in timestamps:
+            timings["reboot_time"] = (datetime.strptime(timestamps["End"], FMT) -\
+            datetime.strptime(timestamps["Start"], FMT)).total_seconds()
```

Comment:
Indent this line a bit more to avoid confusion with the previous line>

Reply:
Sure, added an extra indent here.